code/game/mecha/combat/neovgre

```diff
@@ -0,0 +1,96 @@
+/obj/mecha/combat/neovgre
+	name = "Neovgre, the Anima Bulwark"
+	desc = "Nezbere's most powerful creation, a mighty war machine of unmatched power said to have ended wars in a single night."
+	icon = 'icons/mecha/neovgre.dmi'
+	icon_state = "neovgre"
+	max_integrity = 500 //This is THE ratvarian superweaon, its deployment is an investment
+	armor = list("melee" = 50, "bullet" = 40, "laser" = 25, "energy" = 25, "bomb" = 50, "bio" = 100, "rad" = 100, "fire" = 100, "acid" = 100) //Its similar to the clockwork armour albeit with a few buffs becuase RATVARIAN SUPERWEAPON!!
+	force = 50 //SMASHY SMASHY!!
+	internal_damage_threshold = 0
+	step_in = 3
+	pixel_x = -16
+	layer = ABOVE_MOB_LAYER
+	breach_time = 100 //ten seconds till all goes to shit
+	recharge_rate = 100
+	wreckage = /obj/structure/mecha_wreckage/durand/neovgre
+
+/obj/mecha/combat/neovgre/GrantActions(mob/living/user, human_occupant = 0) //No Eject action for you sonny jim, your life for Ratvar!
+	internals_action.Grant(user, src)
+	cycle_action.Grant(user, src)
+	lights_action.Grant(user, src)
+	stats_action.Grant(user, src)
+	strafing_action.Grant(user, src)
+
+/obj/mecha/combat/neovgre/RemoveActions(mob/living/user, human_occupant = 0)
+	internals_action.Remove(user)
+	cycle_action.Remove(user)
+	lights_action.Remove(user)
+	stats_action.Remove(user)
+	strafing_action.Remove(user)
+
+/obj/mecha/combat/neovgre/MouseDrop_T(mob/M, mob/user)
+	if(!is_servant_of_ratvar(user))
```

Comment:
not gonna do a full review right now, but: 
make this tied to a var. Clock cult will likely be removed, but even if it isn't it will only be adminbus, and this thing is gonna be popular for adminbus. Making admins able to use it without setting antag status is a good idea